src/server/pachyderm_test.go

```diff
@@ -6243,6 +6185,14 @@ func TestUnionInput(t *testing.T) {
 			[]string{"bash"},
 			[]string{
 				"cp -r -L /pfs/TestUnionInput* /pfs/out",
+				"for dir in /pfs/out/*",
```

Comment:
I think this could be simplified by removing the inner loop. There is only one file per directory, and the file name is not even being used in the current code.